content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors (ACEis) and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.
+Data from some animal models suggest several but not all ACEis and several ARBs increase ACE2 expression in cells of some organs.
+Clinical studies have not established that ACE2 expression is increased in humans treated with these medications.
+While randomized clinical trials are ongoing, a variety of observational studies have analyzed the relationship between exposure to ACEis or ARBs and outcomes in patients with COVID-19.
+Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACEi at time of admission.[@doi:10.1056/NEJMoa2007621]
+In contrast, the authors did not find evidence of an association between exposure to an ARB at time of hospitalization and death.
+Because observational studies are subject to confounding, the greatly lower odds of death in the patients exposed to ACEis in this study cannot be interpreted as an effect of the ACEi.
```

Comment:
Is there anything that would make you think that folks receiving ARBs might be a different population than those getting ACEis? Pricing differences, pharmacogenomics, regional treatment patterns? This is a question, not a critique on this sentence.

Reply:
Some reasons to predict variation in prescribing these between populations is a high incidence of persistent cough in Chinese patients taking ACE inhibitors [@pmid:8562296] and variation in incidence of a potentially life-threatening side effect called ACE inhibitor-associated angioedema. [@doi:10.1001/archinte.165.14.1637]

Reply:
I changed the wording a little to emphasize that observational studies are not the standard for assessing drug effects.

Reply:
I need to read the paper more thoroughly, but I do not think that it is correct to say that they found odds of death of .33. That would mean a probability of death of 25%. What they said was that there was an odds RATIO of 0.33. That is very different.

Reply:
Remember that 
odds = probability of yes  divided by probability of no
odds ratio = odds in one condition, divided by odds in another

Reply:
They say, " for ACE inhibitor use, the odds ratio was 0.33."  They don't say anything about an odds of .33.  So I think the proposed text is incorrect and needs to be clarified.  Thanks!

Reply:
Thanks, @dziakj1. That's certainly a big difference, and I need to correct that. 👍

Reply:
Is their paper subject the Table 2 Fallacy (@pmid:23371353)? I thought that might be the case reading it, but I would certainly defer to your expertise.